rpm/prebuild.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+curl -LsSf https://www.tarantool.io/release/2/installer.sh | sudo bash
```

Comment:
Shouldn't we use lowest supported tarantool version for building packages? It is 1.10 ATM.

If there are infrastructure problems on some OSes on 1.10, it is ok to use series-2 for them, but I would like this workaround being commented in the code.

Reply:
I've changed installed Tarantool version to 1.10. Using the lowest Tarantool version seems the good practice.